docs/core/entity/notify.md

```diff
@@ -13,7 +13,10 @@ and can be helpful to send notification messages as (but not limited to):
 - a direct message or chat
 - a screen message on a device's LCD display
 
-Unlike a `text` entity the `notify` entity has no state that can be set. Instead it represents the date and time of the last message sent.
+## States
```

Comment:
Side note: It would be good to harmonize the position of this header for all entities. For climate I think it's currently nested under properties.